src/Screens/Main/Chat/Title.tsx

```diff
@@ -41,6 +65,22 @@ const Title: React.FC<Props> = ({ style, onPress, name, buddyId }) => {
           style={styles.userIcon}
         />
         <RN.Text style={styles.name}>{name}</RN.Text>
+        {!isBanned && (
+          <DropDown
+            items={dropdownItems}
+            testID={'main.chat.menu'}
+            tintColor={colors.black}
+          />
+        )}
```

Comment:
Prefer to use ternary operator instead of &&. It is fine in Web React to use && but in RN && can cause crashes as text or numbers cannot be in React Tree. They will crash it. Null, undefined and False are fine as far as I known. But this is a bad habit in RN that should be checked by eslint.